pre-publish.sh

```diff
@@ -1,17 +1,29 @@
 #!/usr/bin/env bash
 
 # Compile the latest version; update ./pkg and readme files in it.
-wasm-pack build --target nodejs
+wasm-pack build --target nodejs --release
 mv ./pkg/schnorrkel_js.js ./pkg/schnorrkel.node.js
+mv ./pkg/schnorrkel_js_bg.js ./pkg/schnorrkel.node.bg.js
+sed -i -e 's/schnorrkel_js_bg/schnorrkel.node.bg/g' ./pkg/schnorrkel.node.js
+./pack-node.sh
 
-wasm-pack build --target browser
+wasm-pack build --target browser --release
 mv ./pkg/schnorrkel_js.js ./pkg/schnorrkel.wp.js
+sed -i -e 's/schnorrkel_js_bg/schnorrkel.wp.bg.js/g' ./pkg/schnorrkel.wp.js
+./pack-wp.sh
+# will most likely cause an error saying wasm blobl is larger than 4KB
+# ./pack-wp-raw.sh
 
-# Fix the name 
-sed -i -e 's/schnorrkel-js/@parity\/schnorrkel-js/g' pkg/package.json
-
-# Run the script to fix node/browser import support
-./pack-node.sh
+# Replace package.json file.
+cp ./package.json ./pkg/package.json
 
 # publish 
-# wasm-pack publish
\ No newline at end of file
+# wasm-pack publish
+
+# replace this at the top of the .wp.js file 
+# import get_wasm from './schnorrkel.wp.bg.js';
+# var wasm = get_wasm()
+# // this line is a gamble...
+# setTimeout(() => wasm = get_wasm(), 500)
```

Comment:
The caller needs to wait for the actual promise result to ensure it has been added.

Reply:
Indeed but then the export functions will be broken (`export` can only be used in the top level code.). Webpack's async `import()` function should be a proper fix for this I believe.

Reply:
Will defintaly not merge this branch until at least this ugly thing is gone but it is good to have it pushed in here so that it can be tested without the need to place this horrble piece of code in npm :D